<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1080</width>
    <height>1920</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="navi__mirror">
   <widget class="QStackedWidget" name="navi_mirror_scenario">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1083</width>
      <height>1920</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="navi_mirror_wait">
     <widget class="QLabel" name="background_wait">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1080</width>
        <height>1920</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="background.qrc">:/background/background.png</pixmap>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="navi_mirror_start">
     <widget class="QLabel" name="background_start">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1080</width>
        <height>1920</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="background.qrc">:/background/background.png</pixmap>
      </property>
     </widget>
     <widget class="QLabel" name="description_start">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>562</y>
        <width>1080</width>
        <height>398</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="description.qrc">:/description/description_start.png</pixmap>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="navi_mirror_listening">
     <widget class="QLabel" name="background_listening">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1080</width>
        <height>1920</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="background.qrc">:/background/background.png</pixmap>
      </property>
     </widget>
     <widget class="QLabel" name="description_listening">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>562</y>
        <width>1080</width>
        <height>398</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="description.qrc">:/description/description_listening.png</pixmap>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="navi_mirror_done">
     <widget class="QLabel" name="background_done">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1080</width>
        <height>1920</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="background.qrc">:/background/background.png</pixmap>
      </property>
     </widget>
     <widget class="QLabel" name="description_done">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>300</y>
        <width>1080</width>
        <height>398</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="description.qrc">:/description/description_done.png</pixmap>
      </property>
     </widget>
     <widget class="QListWidget" name="object_list">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>840</y>
        <width>1080</width>
        <height>600</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
     </widget>
     <widget class="QTableWidget" name="room_list">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>800</y>
        <width>1080</width>
        <height>600</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="rowCount">
       <number>1</number>
      </property>
      <property name="columnCount">
       <number>8</number>
      </property>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>270</number>
      </attribute>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <attribute name="verticalHeaderDefaultSectionSize">
       <number>80</number>
      </attribute>
      <row/>
      <column>
       <property name="text">
        <string>호실</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>담당자</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>이름</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>담당자 번호</string>
       </property>
      </column>
      <column/>
      <column/>
      <column/>
      <column/>
     </widget>
     <zorder>background_done</zorder>
     <zorder>description_done</zorder>
     <zorder>room_list</zorder>
     <zorder>object_list</zorder>
    </widget>
    <widget class="QWidget" name="navi_mirror_map">
     <widget class="QLabel" name="background_map">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1080</width>
        <height>1920</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="background.qrc">:/background/background.png</pixmap>
      </property>
     </widget>
     <widget class="QLabel" name="description_gotohome">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>1600</y>
        <width>1080</width>
        <height>90</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="description.qrc">:/description/description_gotohome.png</pixmap>
      </property>
     </widget>
     <widget class="QLabel" name="map">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>100</y>
        <width>610</width>
        <height>1300</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="qr">
      <property name="geometry">
       <rect>
        <x>660</x>
        <y>100</y>
        <width>370</width>
        <height>370</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="background.qrc"/>
  <include location="description.qrc"/>
  <include location="mic.qrc"/>
  <include location="select.qrc"/>
 </resources>
 <connections/>
</ui>
